packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -157,6 +164,19 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
       return sumOfPercent;
     });
 
+    /**
+     * Include an imaginary placeholder with value equal to
+     * the difference between (maximum of all bar values) and (current bar value)
+     * while calculating sumOfPercent to get correct scalingRatio for relative variant
+     */
+    let val = total === 0 ? 0 : ((total - barValue) / total) * 100;
```

Comment:
This code only needs to run for MultiStackedBarChartVariant.Relative. Could you move the block inside the this.props.variant === MultiStackedBarChartVariant.Relative condition.